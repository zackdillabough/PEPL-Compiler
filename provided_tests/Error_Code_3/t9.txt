POLY A(M) = 1 + 8 M^0 + 4 M^3;
POLY B = 9;
POLY C(N) = 2 N - 4 N N + 9 N N N;
START
INPUT H;
INPUT I;
INPUT J;

C(H);
B(J);
A(5);

D(H);
E(J);
F(5);

G(H);
K(J);
L(5);

M(H);
N(J);
O(5);

C(H);
B(J);
A(5);

1 2 3 4 5 6
