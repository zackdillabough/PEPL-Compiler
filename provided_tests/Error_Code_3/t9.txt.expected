Error Code 3: 13 14 15 17 18 19 21 22 23 